src/main/java/net/authorize/sample/SampleCode.java

```diff
@@ -343,6 +344,9 @@ private static void RunMethod(String methodName)
                 break;
             case "GetAnAcceptPaymentPage":
             	GetAnAcceptPaymentPage.run(apiLoginId, transactionKey, amount);
+                break;
+			case "CreateChasePayTransaction":
+            	CreateChasePayTransaction.run(apiLoginId, transactionKey, amount);
```

Comment:
Fix indentation in these lines

Reply:
I have made the changes and I have pushed.